components/aboutUs/Contributors.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from "react";
+import styled from 'styled-components';
+
+import {
+  SubInfo,
+  Headline,
+  MidGreenBar,
+  HeadingGroup,
+} from "./styles";
+
+const Wrapper = styled(SubInfo)`
+  display: block;
+  max-width: 990px;
+  margin: 0 auto;
+`;
+
+const ContributionsWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const ContributorBubble = styled.a`
+  display: flex;
+  width: 60px;
+  height: 60px;
+  margin: 8px 9px;
+  border-radius: 50%;
+  object-fit: cover;
+  overflow: hidden;
+`;
+
+const Contributors = () => {
+  const [contributors, setContributors] = useState<any[]>([]);
+
+  useEffect(() => {
+    fetch("https://api.github.com/repos/chingu-voyages/moonshot-chingu-quiz/contributors")
+      .then(response => response.json())
+      .then(json => setContributors(json));
+  }, []);
+
+  return contributors && contributors.length > 0 ? (
```

Comment:
Note: I opted to hide the entire section of no contributors were found, or if the request failed.. better than showing an empty "contributors" section.

Reply:
Great idea, hehe